README.md

```diff
@@ -1,5 +1,9 @@
 # DevOps-Apprenticeship-Notes
+k-6-youtube-workshops
+* Here is the docker post-install instructions that reduce the number of sudo commands: https://docs.docker.com/engine/install/linux-postinstall/
```

Comment:
Nice job adding this link. That definitely helped us out when we were going through the initial install.